package.json

```diff
@@ -18,6 +18,7 @@
     "@storybook/addon-a11y": "^6.0.26",
     "@storybook/addon-essentials": "^6.0.26",
     "@storybook/addon-storysource": "^6.0.26",
+    "@storybook/addon-viewport": "^6.0.26",
```

Comment:
This is necessary because we use the dependency directly at https://github.com/kufu/smarthr-ui/blob/support-reactv17/.storybook/preview.js.
The addon is installed via `@storybook/addon-essentials` but it isn't guaranteed whether the addon is placed in the root `node_modules/` directory.
In this PR, the addon has been installed in the `@storybook/addon-essentials`'s `node_modules/` directory, which breaks our build for storybook.
To fix this, I've installed `@storybook/addon-viewport` explicitly.